tests/performance_tests/CMakeLists.txt

```diff
@@ -7,12 +7,14 @@ configure_file(nodeos_log_2_0_14.txt.gz nodeos_log_2_0_14.txt.gz COPYONLY)
 configure_file(nodeos_log_3_2.txt.gz nodeos_log_3_2.txt.gz COPYONLY)
 configure_file(genesis.json genesis.json COPYONLY)
 configure_file(validate_nodeos_plugin_args.py validate_nodeos_plugin_args.py COPYONLY)
+configure_file(cpuTrxData.json cpuTrxData.json COPYONLY)
 configure_file(userTrxDataTransfer.json userTrxDataTransfer.json COPYONLY)
 configure_file(userTrxDataNewAccount.json userTrxDataNewAccount.json COPYONLY)
 
-add_test(NAME performance_test_basic COMMAND tests/performance_tests/performance_test_basic.py -v -p 1 -n 1 --target-tps 20 --tps-limit-per-generator 10 --test-duration-sec 5 --clean-run WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+add_test(NAME performance_test_basic COMMAND tests/performance_tests/performance_test_basic.py -v -p 1 -n 1 --target-tps 20 --tps-limit-per-generator 10 --clean-run WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
Is there a reason for removing the `--test-duration-sec`?  It was configured that way to decrease the running time of these tests.  Might consider adding that to the `performance_test_basic_ex_cpu_trx_spec` test as well.

Reply:
Please revert removal of `--test-duration-sec 5 `